Eat_and_Greet/src/screens/PostEvent/index.tsx

```diff
@@ -51,22 +107,25 @@ export default function PostEvent() {
             top={"-30%"}
             height={"20%"}
             width={"90%"}
+            onTextChange={setEventDate}
           />
           <TextBox
             placeholder="Enter Start Time..."
             top={"-38%"}
             height={"20%"}
             width={"90%"}
+            onTextChange={setStartTime}
           />
           <TextBox
             placeholder="Enter End Time..."
             top={"-46%"}
             height={"20%"}
             width={"90%"}
+            onTextChange={setEndTime}
           />
         </View>
       </SafeAreaView>
-      <RoundedButton name="Post Event" top="-2%" />
+      <RoundedButton name="Post Event" top="-2%" onPress={addPostToDB} />
```

Comment:
Overall love the change, not going to approve until code smells are way lower than 12. Great job Rob!